lists/br/cnpj.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": {
+    "en": "National Registry of Legal Entities",
+    "local": "Cadastro Nacional da Pessoa Jurídica"
+  },
+  "url": "http://cnpj.info/",
+  "description": {
+    "en": "The CNPJ is the national tax identification number issued by the Ministério da Fazenda, for all non-personal legal entities in Brazil.\nOrganizations register exclusively on the Federal Revenue website. After filling in the necessary data online, relevant documents are sent by mail or presented in person.\nIt is a fourteen digit number that must be included on all receipts and invoices. Tax authorities assign an eight-digit company registration number that makes up the first part of the CNPJ. The following four digits refer to the business place or branch. Company headquarters are usually assigned the number 0001 for this section of the identification number, with other branches receiving different numbers. The final two numbers are check digits."
```

Comment:
I think that the initials CNPJ can refer to both the registry and the tax identification number. I think we should be describing the list first and so I'd probably edit the start of this to "The CNPJ is the national registry that records tax identification numbers issued by the Ministério da Fazenda"